lib/chat-setting-limits.ts

```diff
@@ -157,6 +157,12 @@ export const CHAT_SETTING_LIMITS: Record<LLMID, ChatSettingLimits> = {
     MAX_TOKEN_OUTPUT_LENGTH: 4096,
     MAX_CONTEXT_LENGTH: 128000
   },
+  "gpt-4o-mini": {
+    MIN_TEMPERATURE: 0.0,
+    MAX_TEMPERATURE: 2.0,
+    MAX_TOKEN_OUTPUT_LENGTH: 4096,
```

Comment:
By the way `MAX_TOKEN_OUTPUT_LENGTH: 4096` will also limit gpt-4o mini maximum response length to 4k tokens, it should be 16383 tokens

Reply:
just updated it! thanks 🙌